Samples/iOS-Swift/iOS-SwiftUITests/UIEventTests.swift

```diff
@@ -0,0 +1,41 @@
+import XCTest
+
+class UIEventTests: XCTestCase {
+    private let app: XCUIApplication = XCUIApplication()
+
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        XCUIDevice.shared.orientation = .portrait
+        app.launchEnvironment["io.sentry.ui-test.test-name"] = name
+        app.launch()
+
+        waitForExistenceOfMainScreen()
+    }
+
+    override func tearDown() {
+        app.terminate()
+        super.tearDown()
+    }
+
+    func testUIEventIgnoreTextFieldChange() {
+        app.buttons["Extra"].tap()
+        app.buttons["UI event tests"].tap()
+
+        let label = app.staticTexts["lbBreadcrumb"]
```

Comment:
`l`: To follow the naming pattern of the buttons below and also the other UI tests. 
```suggestion
        let label = app.staticTexts["breadcrumbLabel"]
```